src/stack.js

```diff
@@ -1,29 +1,21 @@
 'use strict'
 
 export default class Stack {
-  // your code here
-  // always add and remove to the back of the array
-  // let stack = [];
-  // to push data
-  // stack.push(a)
-  // stack.push(b)
   constructor(){
     this.arr = []
   }
 
-  add(element) {
-    this.arr.push(element)
-  }
+    add(element) {
+      return this.arr.push(element)
```

Comment:
Also, you should consider using expected names for stack methods. Typically, you would `push` an element to a stack, not `add` it.